spec/license_spec.rb

```diff
@@ -28,7 +28,7 @@
 
       context 'industry approval' do
         it 'should be approved by OSI or FSF or OD' do
-          expect(approved_licenses).to include(spdx_lcase), 'See https://git.io/vzCTV.'
+          expect(approved_licenses).to include(spdx_lcase), 'See https://github.com/github/choosealicense.com/blob/master/CONTRIBUTING.md#adding-new-licenses.'
```

Comment:
Avoid another redirect:

```suggestion
          expect(approved_licenses).to include(spdx_lcase), 'See https://github.com/github/choosealicense.com/blob/gh-pages/CONTRIBUTING.md#adding-new-licenses.'
```

Reply:
avoid master->gh-pages redirect